docs/pages/development/installation.md

```diff
@@ -87,7 +93,7 @@ Additionally, ensure your project's `minSdkVersion` and Gradle version are at le
 
 ## 2. Basic configuration
 
-To scan a QR code to load your project's JavaScript in your development build, you'll need to configure a deep link scheme for your app. The fastest way to do this is with the `uri-scheme` package:
+Development builds use deep links to open projects from the QR code. If you have added a custom deep link scheme to your project, your development build will use it. However, if this isn't the case, you need to configure the deep link support for your app. The `uri-scheme` package will do this for you once you have chosen a scheme.
```

Comment:
```suggestion
Development builds use deep links to open projects from the QR code. If you have added a custom deep link scheme to your project, your development build will use it. However, if this isn't the case, you need to configure deep link support for your app. The `uri-scheme` package will do this for you once you have chosen a scheme.
```

Reply:
Also wonder if it's worth suggesting a scheme or linking to some other documentation for users who might not know what this is?